messaging/src/main/java/org/axonframework/eventhandling/SimpleEventBus.java

```diff
@@ -72,6 +72,12 @@ public Builder spanFactory(@Nonnull SpanFactory spanFactory) {
             return this;
         }
 
+        @Override
```

Comment:
You didn't change the JavaDoc of the `Builder` class and `builder()` method in accordance with this change.

Reply:
Adjusted